src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -58,11 +58,26 @@ UserInfoPopup::UserInfoPopup()
         // items on the right
         auto vbox = head.emplace<QVBoxLayout>();
         {
-            auto name = vbox.emplace<Label>().assign(&this->ui_.nameLabel);
+            auto name_box = vbox.emplace<QHBoxLayout>();
+            name_box.withoutMargin();
+
+            auto name = name_box.emplace<Label>().assign(&this->ui_.nameLabel);
+
+            LayoutCreator<EffectLabel2> copy_user_name = name_box.emplace<EffectLabel2>(this);
+            copy_user_name->getLabel().setText("Copy");
+            copy_user_name->getLabel().setCentered(true);
 

```

Comment:
You can change color of the text with copy_user_name->getLabel().setStyleSheet("color: #5B5B5B"); or copy_user_name->getLabel().setStyleSheet("color: rgb(91,91,91)"); if you prefer the RGB notation.

I think the gray is a bit hard to read tbh. Maybe try some other colors:
![ia16u3nBLl](https://user-images.githubusercontent.com/16636423/63545260-409df180-c527-11e9-99cf-314f40cbcacb.jpg)

Reply:
How about this look?

![green copy button](https://i.nuuls.com/6ARTT.png)